articles/azure/azs-how-create-availability-set.md

```diff
@@ -0,0 +1,163 @@
+---
+title: How to create a availability set | UKCloud Ltd
+description: Creating a availability set in UKCloud for Microsoft Azure
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: 
+toc_sub3:
+toc_sub4:
+toc_title: Create a availability set
+toc_fullpath: Users/How To/azs-how-create-availability-set.md
+toc_mdlink: azs-how-create-availability-set.md
+---
+
+# Overview
+
+This guide will cover how to increase the availability and reliability of your virtual machines on Azure Stack using **availability sets**. availability sets ensure that VMs you deploy on Azure Stack are evenly distributed across multiple physical nodes in a cluster. Doing this ensure that, if a hardware or software failure happens within Azure Stack, only a subset of your VMs are impacted and your overall solution remains available and operational.
```

Comment:
availability sets at the beginning of the second sentence needs an uppercase A

Reply:
Actually, I'd change the structure of this a bit. The second couple of sentences here are just repeated below, so I'd get rid of the "Availability sets" heading and just pull all of the content into the "Overview" section. I also think it still needs a bit of rewording. How about something like:

## Overview

This article shows you how to increase the availability and reliability of your Azure Stack applications using availability sets.

An availability set groups virtual machines (VMs) together so that Azure Stack can control where they are deployed to ensure availability. VMs in the same availability group are deployed across multiple physical nodes within a cluster so that, in the event of a hardware or software failure, only a subset of those VMs are affected and your application can continue to run. If you're building critical applications that must remain available, using availability sets is an important part of ensuring that availability.

For example, if your application has four web server VMs, you can put these in an availability set to ensure that they are deployed across different nodes. If the hardware that one of your web servers is running on experiences a failure, because the other instances are deployed in different nodes, they will remain running and serving your application. You could create a separate availability set for any database-hosting VMs to similarly protect the back end of your application.

Reply:
done